pkg/controllers/dynakube/oneagent/daemonset/arguments.go

```diff
@@ -57,3 +64,7 @@ func (dsInfo *builderInfo) appendProxyArg(argMap *prioritymap.Map) {
 func (dsInfo *builderInfo) hasProxy() bool {
 	return dsInfo.dynakube != nil && dsInfo.dynakube.NeedsOneAgentProxy()
 }
+
+func appendHostIdSource(argMap *prioritymap.Map, hostIdSource string) {
```

Comment:
I don't see any benefits of using function for 1 line to replace one line .

Reply:
fixed in [37f5af7](https://github.com/Dynatrace/dynatrace-operator/pull/2508/commits/37f5af762bc2c0bdef231ee7781663376055dbed)